### PR TITLE
Added important note for issue #4836

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #4836

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/bitlocker/bitlocker-overview.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ A computer with a TPM must also have a Trusted Computing Group (TCG)-compliant B
 
 The system BIOS or UEFI firmware (for TPM and non-TPM computers) must support the USB mass storage device class, including reading small files on a USB flash drive in the pre-operating system environment.
 
+> [!IMPORTANT]
+> You can use an OS drive encrypted without TPM and USB stick, from windows 7, [check procedure here](https://blogs.technet.microsoft.com/tip_of_the_day/2014/01/22/tip-of-the-day-bitlocker-without-tpm-or-usb/).
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-overview.md]:
```suggestion
> From Windows 7, you can encrypt an OS drive without a TPM and USB flash drive. For this procedure, see [Bitlocker without TMP or USB](https://blogs.technet.microsoft.com/tip_of_the_day/2014/01/22/tip-of-the-day-bitlocker-without-tpm-or-usb/).
```

**Comment by mypil** [comment]
@j0rt3g4 - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by j0rt3g4** [comment]
@mypil It's done

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@j0rt3g4 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by j0rt3g4** [comment]
@e0i  Ok I will do the corrections

File: windows/security/information-protection/bitlocker/bitlocker-overview.md [hunk 2]
```diff
@@ -60,6 +60,9 @@ A computer with a TPM must also have a Trusted Computing Group (TCG)-compliant B
 
 The system BIOS or UEFI firmware (for TPM and non-TPM computers) must support the USB mass storage device class, including reading small files on a USB flash drive in the pre-operating system environment.
 
+> [!IMPORTANT]
+> From Windows 7, you can encrypt an OS drive without a TPM and USB flash drive. For this procedure, see [Tip of the day: Bitlocker without TMP or USB](https://blogs.technet.microsoft.com/tip_of_the_day/2014/01/22/tip-of-the-day-bitlocker-without-tpm-or-usb/).
```
Comment (mypil) [inline windows/security/information-protection/bitlocker/bitlocker-overview.md]:
```suggestion
> From Windows 7, you can encrypt an OS drive without a TPM and USB flash drive. For this procedure, see [Tip of the Day: Bitlocker without TPM or USB](https://blogs.technet.microsoft.com/tip_of_the_day/2014/01/22/tip-of-the-day-bitlocker-without-tpm-or-usb/).
```

**Comment by mypil** [comment]
@DulceMontemayor - PR has been copy edited and is ready for final review. This is based on issue #4836.

Thank you.

**Comment by j0rt3g4** [comment]
Any updates on this one?

**Comment by illfated** [comment]
It looks like the document author is quite busy elsewhere, so it may take some more time before the author replies.
Maybe ask the document manager to review & merge this instead?

**Comment by mypil** [comment]
@Dansimp - Can you possibly review and merge this PR?

Thanks you.

**Pull request merged**